gradle.properties

```diff
@@ -5,7 +5,7 @@ cordaProductVersion = 5.2.0
 # NOTE: update this each time this module contains a breaking change
 ## NOTE: currently this is a top level revision, so all API versions will line up, but this could be moved to
 ##   a per module property in which case module versions can change independently.
-cordaApiRevision = 34
+cordaApiRevision = 999
```

Comment:
Is this mistake?

Reply:
its easier when merging down from release when in dev